media/js/privacy/redirects.es6.js

```diff
@@ -0,0 +1,17 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+if (window.location.hash === '#health-report') {
```

Comment:
needs tests

Reply:
Our existing test scripts are only set up to test redirects at the Django level. So, this needs to be a functional test. I'll work on adding a Playwrite one.